src/TaskTupleAwaiter/TaskTupleExtensions.cs

```diff
@@ -1668,6 +1668,1317 @@ private readonly
 	}
 	#endregion
 
+	#region (Task<T1>..Task<T11>)
+	/// <summary>
+	/// </summary>
+	/// <typeparam name="T1"></typeparam>
+	/// <typeparam name="T2"></typeparam>
+	/// <typeparam name="T3"></typeparam>
+	/// <typeparam name="T4"></typeparam>
+	/// <typeparam name="T5"></typeparam>
+	/// <typeparam name="T6"></typeparam>
+	/// <typeparam name="T7"></typeparam>
+	/// <typeparam name="T8"></typeparam>
+	/// <typeparam name="T9"></typeparam>
+	/// <typeparam name="T10"></typeparam>
+	/// <typeparam name="T11"></typeparam>
+	/// <param name="tasks"></param>
+	/// <returns></returns>
```

Comment:
@buvinghausen Aside, I'd like to remove empty docs sometime in favor of `#pragma` suppressions for all the surface area intended for compiler consumption rather than user consumption.

Reply:
Can you steward those changes? I just got back from vacation and am buried with things from work